src/Controls/samples/Controls.Sample.UITests/Issues/Issue19509.xaml.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+	[Issue(IssueTracker.Github, 19509, "Entry TextColor property not working when the Text value is bound after some time", PlatformAffected.iOS)]
+	public partial class Issue19509 : ContentPage
+	{
+		string _text;
+
+		public Issue19509()
+		{
+			InitializeComponent();
+
+			BindingContext = this;
+		}
+
+		protected override async void OnAppearing()
+		{
+			base.OnAppearing();
+
+			await Task.Delay(1500);
```

Comment:
What if we trigger the text change just with a button click instead of a delay? That might speed up things and is less arbitrary.

Reply:
I did as you suggested and it does the job!